pkg/plugins/assign/assign.go

```diff
@@ -84,6 +86,10 @@ func handleGenericComment(pc plugins.Agent, e github.GenericCommentEvent) error
 	return err
 }
 
+func isTeamLogin(login string) bool {
+	return strings.Contains(login, "/")
```

Comment:
This will also include the case `something/` which is non-valid. Let's try to check the length of the slice if you split it by `/` or even better match a regex expression

Reply:
Thanks!! I fixed it to check if it's empty (https://github.com/kubernetes-sigs/prow/pull/164/commits/7440b238f87c02402755b1390b31154410e0ac87)